SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -313,91 +282,81 @@ UnitTest {
 	// private - use TestYourClass.run
 
 	run { | reset = true, report = true|
-		var function;
-		if(reset) { this.class.reset };
-		if(report) { ("RUNNING UNIT TEST" + this).inform };
-		this.class.forkIfNeeded {
-			this.findTestMethods.do { |method|
-				this.setUp;
-				currentMethod = method;
-				//{
-				this.perform(method.name);
-				// unfortunately this removes the interesting part of the call stack
-				//}.try({ |err|
-				//	("ERROR during test"+method).postln;
-				//	err.throw;
-				//});
-
-				this.tearDown;
-			};
-			if(report) { this.class.report };
-			nil
-		};
-
+		this.class.run
```

Comment:
if this is nominally private and redirects to the class method, I think we should deprecate it. There is absolutely no reason to use this now and it will only bring confusion.

Reply:
yes, I thought so too.